cli/channels.go

```diff
@@ -45,9 +45,15 @@ var cmdChannels = []cobra.Command{
 				logUsage(cmd.Use)
 				return
 			}
+			pageMetadata := mfxsdk.PageMetadata{
+				Name:     "",
```

Comment:
Name is missing here

Reply:
Yes. Similar conversation [here](https://github.com/mainflux/mainflux/pull/1589#discussion_r881846268)

Reply:
Okay. No problem. I will work on another PR to add the changes

Reply:
Implementing with this spec in mind
```sh
mainflux-cli users get [all | id <user_id> | email <email_address> | metadata <metadata>] <user_auth_token>
```